### DISCUSSION TITLE
Multiple Stations and Sharing Keys

### DISCUSSION THREAD

**Discussion by tkreyche** [jvde-github/AIS-catcher#309] [General]
Dear Jasper and AIS Community,

I'm planning to add additional Raspberry Pi based stations. One of this will be at the marina where I keep my boat, which is about 5 miles away from my house, where I have my current station. It will have better coverage of a certain area due to hills between the two locations.

Another station I am considering will be a second station at my house with a highly directional antenna to better cover a different area. The current station at my house uses an omni-directional antenna.

I'm also planning to add a station on the opposite coast, which is a couple thousand miles away.

So the question is whether each station should have its own sharing key, regardless of location or if the same sharing key can or should be used for any or all stations.

Thanks, Tom

**Comment by jvde-github** [comment]
Hi Tom,

great! 

It is really your option, you can use the same key and then the data from the stations is aggregated in one dashboard. You can also just create two new stations and keep it separate (so the stats reflect the characteristics of the antennas). 

As a side note, if you have three instances of AIS-catcher running locally you can start a 4th one that aggregates for you your local data (either via UDP or TCP). 

Cheers, Jasper

**Reply by tkreyche** [reply]
Thanks!
